app/src/main/java/com/android/solvit/shared/model/chat/ChatRepository.kt

```diff
@@ -8,6 +8,15 @@ interface ChatRepository {
       receiverUid: String
   )
 
+  fun linkChatToRequest(
+      chatRoomId: String,
+      serviceRequestId: String,
```

Comment:
The purpose here is just creating a firestore document that maps a chatroom id to a service request id, it would be inefficient to pass the whole request as a parameter